### PR TITLE
Navbar

### LINKED ISSUE #47: Make navbar component
- Logout button/link
- Link to home page
- Responsiveness

### PR DESCRIPTION
**Pull request by mmoila**
- Add navbar to app layout
- AuthProvider removed as it is not necessarily required

Closes #47

### CODE REVIEW

File: src/lib/auth.ts [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+    }),
+  ],
+};
 
-const handler = NextAuth(authOptions)
+const handler = NextAuth(authOptions);
 
-export { handler as GET, handler as POST }
\ No newline at end of file
+export { handler as GET, handler as POST };
```
Comment (piamanns) [inline src/lib/auth.ts]: This sequence can be deleted entirely. The handler is exported in src/app/api/auth/[...nextauth]/route.ts (which is the correct place)

Comment (piamanns) [inline src/lib/auth.ts]: This sequence can be deleted entirely (it's a duplicate in the original code). The handler is exported in src/app/api/auth/[...nextauth]/route.ts, as it should be.

**Review by piamanns — APPROVED** [review]
Looks good!

**Review by anketola — APPROVED** [review]
Good job. Everything seems to be fine. We can add the SessionProvider later if it's needed.

File: src/lib/auth.ts (hunk 1 above)
Comment (mmoila) [reply src/lib/auth.ts]: Removed duplicate code

**Pull request merged**